tests/s25Main/integration/testTrading.cpp

```diff
@@ -26,6 +24,7 @@
 #include "gameData/JobConsts.h"
 #include <rttr/test/LogAccessor.hpp>
 #include <boost/test/unit_test.hpp>
+#include <boost/variant/variant.hpp>
```

Comment:
where is this variant used?

Reply:
in the call to `this->TradeOverLand(players[0]->GetHQPos(), GD_BOARDS, 2);`
The `GD_BOARDS` implicitely constructs a variant, but it is only forward declared in the header, so users need to include it. This is done because the variant header is huge.